workflows/cloudify_system_workflows/snapshot.py

```diff
@@ -263,21 +283,64 @@ def _update_es_node(es_node):
                 }
 
 
-def _restore_elasticsearch(ctx, tempdir, es):
+def _restore_elasticsearch(ctx, tempdir, es, metadata):
     ctx.send_event('Deleting all ElasticSearch data')
     elasticsearch.helpers.bulk(
         es,
         _clean_up_db_before_restore(es, ctx.execution_id))
     es.indices.flush()
 
-    def es_data_itr():
+    cloudify_events = es.indices.exists(index='cloudify_events')
+    from_cloudify_events = metadata[_M_HAS_CLOUDIFY_EVENTS]
+
+    # cloudify_events -> cloudify_events, logstash-* -> logstash-*
+    def get_data_itr():
         for line in open(os.path.join(tempdir, _ELASTICSEARCH), 'r'):
             elem = json.loads(line)
             _update_es_node(elem)
             yield elem
 
+    # logstash-* -> cloudify_events
+    def l_to_ce():
+        for elem in get_data_itr():
+            if elem['_index'] != 'cloudify_storage':
+                elem['_index'] = 'cloudify_events'
```

Comment:
Let's make cloudify_storage and cloudify_events some constants at the top of this file